packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -215,6 +215,10 @@ export default class ProjectFile {
       manager === 'trufle' ? { manager: 'truffle' } : pickBy({ ...this.data.compiler, ...configOptions });
   }
 
+  public translateMaybeAlias(nameOrAlias: string): string {
```

Comment:
While accurate, I'm not sure this name is immediately clear. How about `contractOrAlias`?